Assets/XML/Units/CIV4PromotionInfos.xml

```diff
@@ -4184,7 +4184,7 @@
 			<bShiftDown>0</bShiftDown>
 			<bCtrlDown>0</bCtrlDown>
 			<iHotKeyPriority>0</iHotKeyPriority>
-			<Button>,Art/Interface/Buttons/Promotions/Logistics1.dds</Button>

```

Comment:
Are you sure this is necessary? The buttons do work in the game don't they? I think I know of crashes/loading problems without the leading comma in some systems. (Same for below.)